### ISSUE TITLE
use-32bit-slots feature is broken

### ISSUE THREAD

**Issue by Veykril** [Veykril/wasm3-rs#12]
```rust
#[no_mangle]
pub extern "C" fn add_u32(foo: u32, bar: u32) -> u32 {
    foo + bar
}
```
This function returns the correct result when using 64bit slots, but an incorrect result when using 32bit slots.

**Comment by Veykril** [comment]
@ericflo, this might be a cause for your problem as well since this feature is enabled by default. Can you check whether your problem persists if you disable default features?

**Comment by Veykril** [comment]
Should be fixed in #56858c8.
